NuGet.config

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
+    <add key="dotnet3.1" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1/nuget/v3/index.json" />
```

Comment:
Added so we can target a 3.1 preview runtime.

Reply:
I don't think this is something we should worry about or support. I would only care about support 6.0 preview runtimes right? I assume we want to be out of the business of supporting preview runtimes?